### PR TITLE
Prepare pymesync for PyPI

### PR DESCRIPTION
**Pull request by matthewrsj**
Let's put this baby on PyPI! #96

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+pymesync
+========
+
+Python module for TimeSync
```
Comment (ElijahCaine) [inline README.rst]: We should also add a section in here that you can install via setup.py or pip. This change might happen once it's on pypi.

**Comment by matthewrsj** [comment]
Well, of course all the tests are failing. I moved them.

**Comment by ElijahCaine** [comment]
@Kennric We troubleshooted the installation (it wasn't working for one reason or another) but now it looks good. You should be able to run the following successfully:
```
$ virtualenv venv && source venv/bin/activate
[...]
(venv)$ pip install https://testpypi.python.org/packages/source/p/pymesync/pymesync-0.0.5.tar.gz#md5=7da5e74daf1d41d287509a060519c8d9
[...]
(venv) $ python
[...]
>>> from pymesync import TimeSync  # imports successfully
>>> # do some timesync test in here, I don't remember them off the top of my head
```

**Comment by matthewrsj** [comment]
Verbal +1 from @Kennric

**Pull request merged**